articles/quickstarts/index.md

```diff
@@ -19,17 +19,146 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
+## Installation options
 
-To proceed with installing the QDK and creating Q# projects, select your preferred setup:
+You can use the QDK three ways:
 
-[Develop with Q# applications](xref:microsoft.quantum.install.standalone) - Choose this approach to work with Q# from the command prompt. This does not require a driver or a host program like the below options.
+- [Install the QDK locally](#install-the-qdk-locally)
+- [Use the QDK online](#use-the-qdk-online)
+- [Use a QDK Docker image](#use-the-qdk-with-docker)
 
-[Develop with Q# Jupyter Notebooks](xref:microsoft.quantum.install.jupyter) - Select this environment to run Q# code in cells with embedded text or create quantum computing interactive tutorials. 
+## Install the QDK locally
 
-[Develop with Q# and Python](xref:microsoft.quantum.install.python) - Enables you to combine Python and Q# to create a Python host program that calls Q# operations.
+You can develop Q# code in most of your favorites IDEs, as well as integrate Q# with other languages such as Python and .NET (C#, F#).
 
-[Develop with Q# and .NET](xref:microsoft.quantum.install.cs) - Combine C#, F#, or VB.NET with Q# to create a .NET host program that calls Q# operations.
+| | **VS Code**| **Visual Studio** | **Jupyter Notebooks** | **Command line**|
+|:-----|:-----:|:-----:|:-----:|:-----:|
+|**OS** |Cross-platform |Windows only |Cross-platform |Cross-platform |
+|**Q# standalone** |&#10004; | &#10004; | &#10004; | &#10004; |
+|**Q#  plus Python** |&#10004; |&#10006; |&#10004; |&#10004; |
+|**Q# plus .NET (C#, F#)**|&#10004; |&#10004;|&#10006; |&#10004; |
+
+### Prerequisites 
+
+- [.NET Core SDK 3.1 or later](https://www.microsoft.com/net/download)
+
+### Configure for VS Code
+
+To configure VS Code:
+
+1. Download and install [VS Code](https://code.visualstudio.com/download) (Windows, Linux and Mac).
+2. Install the [Microsoft QDK for VS Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode).
+
+### Configure for Visual Studio
+
+To configure Visual Studio:
+
+1. Download and install [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3 or greater, with the .NET Core cross-platform development workload enabled.
+2. Download and install the [Microsoft QDK](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit).
+
+### Configure for Jupyter Notebooks and Python
+
+Jupyter Notebooks allow running in-place code alongside instructions, notes, and other content. This environment is ideal for writing Q# code with embedded explanations or quantum computing interactive tutorials. Here's what you need to do to start creating your own Q# notebooks.
+
+> [!NOTE]
+> In Q# Jupyter Notebooks, you can only run Q# code, and the operations cannot be called from external host programs (e.g. Python or C# files). This environment is not appropriate if your goal is to combine an external classical host program with the quantum program.
```

Comment:
Since this is just the install page, I took out this note. We do link to the Ways to Run a Q# Program topic in the Next Steps section - that might be a better place to add the optional workflows.